scripts/markdown-link-check.sh

```diff
@@ -35,7 +35,7 @@ ${DOCKER} run ${DOCKER_RUN_OPTS} --rm -t \
   -w /github/workspace \
   --entrypoint /usr/bin/find \
   docker.io/robertbeal/markdown-link-checker \
-  website \( -type f -name "*.md" -or -name "*.markdown" \) -exec markdown-link-check --config .markdownlinkcheck.json --quiet --verbose {} \; \
+  find docs/resources docs/data-sources \( -type f -name "*.md" -or -name "*.markdown" \) -exec markdown-link-check --config .markdownlinkcheck.json --quiet --verbose {} \; \
```

Comment:
This should resolve the script error -- when Docker's `--entrypoint` flag is given, any of the arguments passed at the end go directly to the `--entrypoint` command. So in this case the `find` here is redundant and why the output contains:

```
find: find: No such file or directory
```

Which ultimately shows up to the `markdown-link-check.sh` script as an error. 👍 

```suggestion
  docs/resources docs/data-sources \( -type f -name "*.md" -or -name "*.markdown" \) -exec markdown-link-check --config .markdownlinkcheck.json --quiet --verbose {} \; \
```

Running this script locally with this change, I get:

```
./scripts/markdown-link-check.sh docker "" rw $(pwd)
==> Checking Markdown links...
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

FILE: docs/resources/release.md

FILE: docs/data-sources/template.md
==================> MARKDOWN LINK CHECK SUCCESS <==================

[✔] All links are good!

===================================================================
```